src/index.ts

```diff
@@ -48,7 +54,7 @@ const usePactGet = (alias: string) => {
   const testCaseTitle = Cypress.currentTest.title
   formattedAlias.forEach((alias) => {
     cy.get(alias).then((response: any) => {
-        console.log(response)
+      console.log(response)
```

Comment:
should we make this conditional so that we only get it for verbose/debug modes?

apologies as this comment isn't directly related to this PR 👍🏾